src/routes/settings/assistants/[assistantId]/+page.server.ts

```diff
@@ -21,6 +22,30 @@ async function assistantOnlyIfAuthor(locals: App.Locals, assistantId?: string) {
 	return assistant;
 }
 
+export async function load({ parent, params, locals }) {
+	const data = await parent();
```

Comment:
Note that this triggers the root's `+layout.server.ts`'s `load`.

So each time we switch assistant, this will trigger a load of _all assistants_

IMO we can get the redirect in `page.ts` - that way the load isn't triggered - since the frontend will be able to execute it. 

---

TLDR:

- Keep `page.ts` as is
- remove the redirect / `parent()` call here
- The rest of the PR is great!

Reply:
handled in 1d5994fdca8a9b9ab9aae15cca2e5e1da884c84b